### ISSUE TITLE
relative pronoun vs article discrepancy

### ISSUE THREAD

**Issue by e-ehrhardt** [morphgnt/sblgnt#26]
I think a word can't be marked both RA if it is the relative pronoun, right?

100213 RA ----NPM- οἵ οἵ οἵ ὅς

**Comment by jtauber** [comment]
Correct. The lemmatization and part of speech are inconsistent but it isn't clear which way to correct to.

**Comment by jtauber** [comment]
@emg what do you think?
